src/applications/mhv-secure-messaging/tests/e2e/sort-filter-test/secure-messaging-advanced-filter-custom-date-custom-folder.cypress.spec.js

```diff
@@ -91,24 +91,24 @@ describe('SM INBOX ADVANCED CUSTOM DATE RANGE SEARCH', () => {
       2,
       1,
     );
-    const currentYear = format(new Date(), 'yyyy');
-    const startMonth = format(subMonths(new Date(), 2), 'MMMM');
-    const endMonth = format(new Date(), 'MMMM');
+    const date = GeneralFunctionsPage.getParsedDate(new Date());
```

Comment:
suggesting to destructure and use as const below
```suggestion
    const { startMonth, endMonth, year } = GeneralFunctionsPage.getParsedDate(new Date());
```